bip-coalesce-wildcard.mediawiki

```diff
@@ -0,0 +1,78 @@
+<pre>
+  BIP: (no number)
+  Title: "Coalescing Transaction" Specification (wildcard inputs)
+  Author: Chris Priest <cp368202@ohiou.edu>
+  Status: Draft
+  Type: Standards Track
+  Created: 2015-11-30
+</pre>
+
+==Abstract==
+
+This specification defines a new type of transaction that supplements (not replaces)
+normal "non coalescing" bitcoin transactions.
+
+==Motivation==
+
+Normal "non-coalescing" Bitcoin Transactions have one large inefficiency: When you want to spend
+from multiple inputs with the exact same scriptPubKey, you have to list each
+input separately, along with the same signature multiple times, even though the signature expresses the same information.
+This bloats the transaction size and makes it expensive to spend from small value inputs.
+
+Because small value inputs are expensive to send, they remain in the UTXO pool
+which full nodes have to keep around. It is believed that long term increase of the UTXO
+set can have negative scaling consequences on the network.
+
+If maximum blocksize is made to increase *slower* than the actual number of transactions bitcoin users are sending
+to the network, this problem is projected to get worse. In other words, as transaction
+fees increase, the minimum economical value of a spending a UTXO will increase.
```

Comment:
Extra "a" in this line:
* the minimum economical value of **a** spending a UTXO